src/CloudSeis.jl

```diff
@@ -706,8 +707,13 @@ function cache_from_file!(io::CSeis{T,N,NotACompressor}, extentindex) where {T,N
     @debug "...data read ($mbps MB/s -- $mb MB)"
     nothing
 end
+function cache_from_file!(io::CSeis{T,N,NotACompressor}, extentindex, regularize) where {T,N}
+    regularize || error("'regularize=false' is not supported for 'NotACompressor'")
+    notacompressor_cache_from_file!(io, extentindex)
+end
 
-function cache_from_file!(io::CSeis{T,N,BloscCompressor}, extentindex) where {T,N}
+function cache_from_file!(io::CSeis{T,N,BloscCompressor}, extentindex, regularize) where {T,N}
+    regularize || error("regularize=false is not supported for 'BloscCompressor'")
```

Comment:
sure, we could change the name. I chose "regularized" rather than "compressed" because this whole concept of storing cache compressed only applies to to the left justify compressor.  I wanted to try to make that clear, and to make it clear that we don't support storing compressed cache for any other type of compression.